R/calcIVarCost.R

```diff
@@ -35,6 +35,7 @@ calcIVarCost <- function() {
   xq <- filter(merged, variable == "Variable non fuel cost")
   xq <- xq %>% select(-c("PRIMES", "variable")) %>%
                rename("variable" = "OPEN.PROM")
+  xq[["unit"]] <- "EUR/MWh"
```

Comment:
the `read` function used above does not deliver units?

Reply:
@giannou The read function mistakenly assigns EUR/KW to the variable cost.